docs/types/expired-certificate.md

```diff
@@ -0,0 +1,29 @@
+# Expired Certificate
+
+A TLS certificate issued to a host at your organization has expired. The certificate has not yet been renewed.
+
+## Problem description
+
+Publically available Certificate Transparency logs contain information on all TLS certificates that have been generated 
+into use. According to those logs, a certificate assigned to a host in your organization has expired, and it has not yet been 
+renewed. When a certificate expires, services relying on TLS on that host no longer work correctly, and those services may 
+not be reachable by their intended users.
+
+## Verifying the issue
+
+You can use a public Certificate Transparency logs search engine such as crt.sh to check the status of your certificate. 
+Replace the hostname in the query string with your hostname. 
+* https://crt.sh/?q=host.example.com
+
```

Comment:
Should we mentio that  it is possible that old expiring certificate is being replaced with wildcard cert. The trouble is, that we can't probably go as far as instructing how the actual situation is verified with different browsers.

Reply:
Added a mention